### PR TITLE
[FIX] l10n_*: Fix test case

### PR DESCRIPTION
**Pull request by jva-odoo**
L10n_ch:

- The 10n_ch_postal_chf field is moved in res.partner.bank with new name l10n_ch_isr_subscription_chf by commit https://github.com/odoo/odoo/commit/f3c1e6cfbe70f9dd37cc7c7b3b3464c5b9af280b
- res.partner.bank created by form tag so all onchange is called and required data is filled

l10n_it_edi:

- field type is renamed with move_type in account.move so change default_move_type instead of default_type

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/l10n_ch/tests/test_l10n_ch_isr.py [hunk 1]
```diff
@@ -83,11 +85,13 @@ def test_isr(self):
         test_account = self.create_account('250097798')
         invoice_1.partner_bank_id = test_account
         self.isr_not_generated(invoice_1)
+        invoice_1.partner_bank_id.write({
+            'l10n_ch_isr_subscription_chf': 'CHF'
```
Comment (yvaucher) [inline addons/l10n_ch/tests/test_l10n_ch_isr.py:89]: This is a wrong value for that field.

This field must contain a ISR subscription number such as `01-162-8`

**Comment by jco-odoo** [comment]
@jva-odoo Can you recheck and rebase the branch to the saas-version where type changes into move_type (probably 13.4)

File: addons/l10n_ch/tests/test_l10n_ch_isr.py (hunk 1 above)
Comment (ppr-odoo) [reply addons/l10n_ch/tests/test_l10n_ch_isr.py:89]: @yvaucher  This test case has been fixed in this commit https://github.com/odoo/odoo/commit/161796e2d9895dfe1508ff233596d2ae95fa5c9f
Thanks

**Comment by ppr-odoo** [comment]
This will be fixed https://github.com/odoo/odoo/pull/54616

**Pull request closed without merging**